client/my-sites/plugins/plugin-eligibility/index.jsx

```diff
@@ -0,0 +1,85 @@
+/**
+ * External dependencies
+ */
+import React, { Component, PropTypes } from 'react';
+import { localize } from 'i18n-calypso';
+import page from 'page';
+
+/**
+ * Internal dependencies
+ */
+import MainComponent from 'components/main';
+import HeaderCake from 'components/header-cake';
+import EligibilityWarnings from 'blocks/eligibility-warnings';
+
+class PluginEligibility extends Component {
+	static propTypes = {
+		pluginSlug: PropTypes.string,
+		siteSlug: PropTypes.string,
+		translate: PropTypes.func,
+	};
+
+	getBackUrl = () => {
+		const { pluginSlug, siteSlug } = this.props;
+
+		return `/plugins/${ pluginSlug }/${ siteSlug }`;
+	};
+
+	goBack = () => page( this.getBackUrl() );
```

Comment:
could we pull this outside of the component and pass it in? It would be far easier to test the component for one if we didn't directly call page. here is _one way_ we could do this:

```js
this.props.navigateTo( this.getBackUrl() );

const withNagivation = Component => props => <Component { ...{ props }, navigateTo: page } />

export default withNavigation( localize( PluginEligibility ) );
```